seahorn/jobs/array_list_sort/aws_array_list_sort_harness.c

```diff
@@ -16,7 +16,11 @@ size_t item_size;
 int compare(const void *a, const void *b) {
     assume(AWS_MEM_IS_READABLE(a, item_size));// first element readable in compare function
     assume(AWS_MEM_IS_READABLE(b, item_size)); //second element readable in compare function
+#ifdef __KLEE__
+    return 0;
```

Comment:
Pl add a comment like the following

```
We want qsort to not modify the passed in array. Since qsort is linked in for klee, we need to override the compare function so that the array remains unmodified.
```